app/helpers/Flags.js

```diff
@@ -0,0 +1,87 @@
+import { assign } from 'lodash';
+import React, { useContext, useEffect, useState } from 'react';
+
+import { buildTimeFlags } from '../constants/flagsEnv';
+import { FEATURE_FLAG_VALS } from '../constants/storage';
+import { GetStoreData, SetStoreData } from './General';
+
+/**
+ * Converts the flag name from env into a more human readable form.
+ *
+ * Example:
+ *
+ * `getCleanedFlagName('hello_world')` becomes `Hello World`
```

Comment:
nit: semi-shouty case is weird: Flags That Have Long Names Look Strange

prefer "Sentence case for flags"